pyro/primitives.py

```diff
@@ -462,7 +498,7 @@ def module(name, nn_module, update_module_params=False):
                         param_name
                     ] = target_state_dict[_name]
                 else:
-                    nn_module._parameters[mod_name] = target_state_dict[_name]
+                    nn_module._parameters[mod_name] = target_state_dict[_name]  # type: ignore[assignment]
```

Comment:
`nn_module._parameters`'s type is `nn.Parameter` and `target_state_dict[_name]` is `torch.Tensor`.